src/pages/en/editor-setup.md

```diff
@@ -37,3 +37,39 @@ In addition to local editors, Astro also runs well on in-browser hosted editors,
 - [StackBlitz](https://stackblitz.com/) and [CodeSandbox](https://codesandbox.io/) - online editors that run in your browser, with built-in syntax highlighting support for `.astro` files. No installation or configuration required!
 - [GitHub.dev](https://github.dev/) - allows you to install the Astro VS Code extension as a [web extension](https://code.visualstudio.com/api/extension-guides/web-extensions), which gives you access to only some of the full extension features. Currently, only syntax highlighting is supported.
 - [Gitpod](https://gitpod.io/) - a full dev environment in the cloud that can install the official Astro VS Code Extension from Open VSX.
+
+## Other tools
+
+### ESLint
+
+[ESLint](https://eslint.org/) is a popular linter for JavaScript and JSX. For Astro support, [a community maintained plugin](https://github.com/ota-meshi/eslint-plugin-astro) can be installed
+
+See [the project's User Guide](https://ota-meshi.github.io/eslint-plugin-astro/user-guide/) for more information on how to install and setup this for your project
+
+### Prettier
+
+[Prettier](https://prettier.io/) is a popular formatter for JavaScript, HTML, CSS and more. To add Astro support (`.astro` files), we created and maintain an [official Prettier Astro plugin](https://github.com/withastro/prettier-plugin-astro)
```

Comment:
What do you think about simplifying the second sentence slightly? I also switched order to “Astro Prettier plugin” as it’s a “Prettier plugin” for Astro, rather than an “Astro plugin”.

```suggestion
[Prettier](https://prettier.io/) is a popular formatter for JavaScript, HTML, CSS and more. To add support for formatting `.astro` files, use [the official Astro Prettier plugin](https://github.com/withastro/prettier-plugin-astro).
```